wagtail_shortcode/static_src/custom.d.ts

```diff
@@ -12,6 +12,13 @@ declare module '*.svg' {
 declare global {
   // Wagtail globals
 
+  interface Window {
```

Comment:
**Note:**
Some of these types aren't exported e.g. `import type { TooltipEntity }  from 'draftail';`, so as not to waste time on resolving this i've defined them as any.

For example, I can't define:
```
const { TooltipEntity } = window.draftail;
```
as
```
import type { TooltipEntity as TooltipEntityType }  from 'draftail';`

const { TooltipEntity } = window.draftail as TooltipEntityType;
```